dispatch/base.h

```diff
@@ -254,4 +254,20 @@
 
 typedef void (*dispatch_function_t)(void *_Nullable);
 
+#ifdef __SIZEOF_POINTER__
```

Comment:
I didn't realize this was in base.h, sorry this is internal stuff and should be in "src/shims/hw_config.h"

for the test that uses it today, just use `__SIZEOF_POINTER__` it's less of an issue for tests.